api/v1alpha1/user_types.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// UserSpec defines the desired state of S3User
+type UserSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
```

Comment:
Please remove boilerplate.

Reply:
Also somewhat relevant : I'm slightly put off by the name difference between "UserSpec" and "S3User". I understand initially the feature was "user management", but then it seemed better from the client point of view to be explicit about the user (as in : `kubectl get s3user` is self-explaining and less prone to error than `kubectl get user`), so I'm in favor of S3User... but would it be possible to rename the other parts accordingly (S3UserSpec, S3UserStatus, ...) ?